src/FirstSteps/RPG/AdventuresRegistry.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FirstSteps.RPG.Adventures;
+using Spectre.Console;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirstSteps.RPG
+{
+    public class AdventuresRegistry 
+    {
+        private static List<IAdventure> _adventures = new List<IAdventure>()
+        {
+          new Dungeons(),
+          new Forest(),
+          new Hell(),
+          new Mine(),
+          new TreasureChest(),
+          new Village(),
+          new Inventory()
+        };
+        
+        public void GetMenu()
+        {
+            var menu = new SelectionPrompt<string>();
+
+            menu.AddChoice("Hero")
+                .AddChild("Stats");
+
+
+            menu.AddChoiceGroup("Adventures", new string[] {"Treasure Chest", "Inventory", "Forest", "Dungeons", "Mine", "Village", "Hell" });
```

Comment:
When you move it to the `Game.cs` you will be able to get all commands from `GetCommands()` method.